upup/pkg/fi/cloudup/bootstrapchannelbuilder_test.go

```diff
@@ -46,6 +46,7 @@ func TestBootstrapChannelBuilder_BuildTasks(t *testing.T) {
 	runChannelBuilderTest(t, "cilium", []string{"dns-controller.addons.k8s.io-k8s-1.12", "kops-controller.addons.k8s.io-k8s-1.16"})
 	runChannelBuilderTest(t, "weave", []string{})
 	runChannelBuilderTest(t, "amazonvpc", []string{"networking.amazon-vpc-routed-eni-k8s-1.12", "networking.amazon-vpc-routed-eni-k8s-1.16"})
+	runChannelBuilderTest(t, "awsiamauthenticator", []string{"dns-controller.addons.k8s.io-k8s-1.12", "kops-controller.addons.k8s.io-k8s-1.16"})
```

Comment:
I think this list of addons should mainly include the authenticator. Try this and then rerun hack/updated-expected.sh:

```suggestion
	runChannelBuilderTest(t, "awsiamauthenticator", []string{"authentication.aws-k8s-1.12"})
```

The goal is to have the authenticator manifest rendered in `upup/pkg/fi/cloudup/tests/bootstrapchannelbuilder/awsiamauthenticator/`

Reply:
Ah, thank you. I wasn't using that hack script correctly. I have a much better understanding of whats happening in there now.